### PR TITLE
Disable pinning version of GH actions

### PR DESCRIPTION
**Pull request by petertrr**

### CODE REVIEW

File: .github/workflows/releasability-check.yml [hunk 1]
```diff
@@ -13,6 +13,6 @@ jobs:
     permissions:
       id-token: write
       contents: write
-    uses: SonarSource/gh-action_release/.github/workflows/releasability-check.yaml@5.3.2
+    uses: SonarSource/gh-action_release/.github/workflows/releasability-check.yaml@5
```
Comment (jonas-wielage-sonarsource) [inline .github/workflows/releasability-check.yml:16]: Should be v5

File: .github/workflows/dogfood.yml [hunk 2]
```diff
@@ -36,14 +36,14 @@ jobs:
...
-        uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # 2.1.0
+        uses: Ilshidur/action-slack@2
         env:
           SLACK_WEBHOOK: ${{ fromJSON(steps.secrets.outputs.vault).SLACK_WEBHOOK }}
           SLACK_OVERRIDE_MESSAGE: 'Dogfood build for `${{ steps.dogfood.outputs.sha1 }}`: *successful*'
         with:
           args: 'Succeed to build dogfood branch'
       - name: Notify failures on Slack
-        uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # 2.1.0
+        uses: Ilshidur/action-slack@2
```
Comment (jonas-wielage-sonarsource) [inline .github/workflows/dogfood.yml:46]: should be V2 I believe

File: .github/workflows/dogfood.yml [hunk 3]
```diff
@@ -36,14 +36,14 @@ jobs:
         run: echo "The dogfood branch was `${{ steps.dogfood.outputs.dogfood-branch }}` and its HEAD SHA1 was `${{ steps.dogfood.outputs.sha1 }}`"
       #slack notifications
       - name: Notify success on Slack
-        uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # 2.1.0
+        uses: Ilshidur/action-slack@2
```
Comment (jonas-wielage-sonarsource) [inline .github/workflows/dogfood.yml:39]: V2

File: .github/workflows/dogfood.yml [hunk 4]
```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
       - name: get secrets
         id: secrets
-        uses: SonarSource/vault-action-wrapper@2.8.0
+        uses: SonarSource/vault-action-wrapper@2
```
Comment (jonas-wielage-sonarsource) [inline .github/workflows/dogfood.yml:22]: V2

**Review by jonas-wielage-sonarsource — APPROVED** [review]
Mostly lgtm, please see my comments

File: .github/workflows/releasability-check.yml (hunk 1 above)
Comment (petertrr) [reply .github/workflows/releasability-check.yml:16]: Git tag is [not](https://github.com/SonarSource/gh-action_release/releases) prefixed with `v` for this action. I'm not sure if Github can deduce it automatically, but it definitely can match tag without a `v`

File: .github/workflows/dogfood.yml (hunk 2 above)
Comment (petertrr) [reply .github/workflows/dogfood.yml:46]: Same here, this action doesn't prefix tags

**Pull request merged**